apps/experiments/views/experiment.py

```diff
@@ -332,14 +338,22 @@ def get_initial(self):
 
 def _get_voice_provider_alpine_context(request):
     """Add context required by the experiments/experiment_form.html template."""
+    exclude_services = [SyntheticVoice.OpenAIVoiceEngine]
+    if flag_is_active(request, "open_ai_voice_engine"):
+        exclude_services = []
     return {
         "form_attrs": {"x-data": "experiment", "enctype": "multipart/form-data"},
         # map provider ID to provider type
         "voice_providers_types": dict(request.team.voiceprovider_set.values_list("id", "type")),
         "synthetic_voice_options": sorted(
             [
-                {"value": voice.id, "text": str(voice), "type": voice.service.lower()}
-                for voice in SyntheticVoice.objects.all()
+                {
+                    "value": voice.id,
+                    "text": str(voice),
+                    "type": voice.service.lower(),
+                    "provider_id": voice.voice_provider.id if voice.voice_provider else None,
```

Comment:
nit: use _id field instead of doing related query:
```suggestion
                    "provider_id": voice.voice_provider_id,
```

Explanation: every foreign key field on a model exposes a `{field_name}_id` property which is the ID of the related object (or None if it's not set).